dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBNoSQL/AzureCosmosDBNoSQLVectorStoreCollectionQueryBuilder.cs

```diff
@@ -59,8 +59,8 @@ public static QueryDefinition BuildSearchQuery<TVector>(
         var queryDefinition = new QueryDefinition(query);
 
         queryDefinition.WithParameter(VectorVariableName, vector);
-        queryDefinition.WithParameter(OffsetVariableName, searchOptions.Offset);
-        queryDefinition.WithParameter(LimitVariableName, searchOptions.Limit);
+        queryDefinition.WithParameter(OffsetVariableName, searchOptions.Skip);
+        queryDefinition.WithParameter(LimitVariableName, searchOptions.Top);
```

Comment:
The consts still have old names.

Reply:
This reflects the underlying datastore naming, so I think this is correct.